posthog/helpers/dashboard_templates.py

```diff
@@ -0,0 +1,164 @@
+    )
+
+    DashboardItem.objects.create(
+        team=dashboard.team,
+        dashboard=dashboard,
+        name="Repeat users over time",
+        type=TRENDS_LINEAR,
+        filters={
+            TREND_FILTER_TYPE_EVENTS: [{"id": "$pageview", "math": "dau", "type": TREND_FILTER_TYPE_EVENTS}],
+            INTERVAL: "day",
+            DATE_FROM: "-30d",
+            SHOWN_AS: TRENDS_STICKINESS,
+        },
+        last_refresh=now(),
+        color=random.choice(DASHBOARD_COLORS),
+    )
+
+    DashboardItem.objects.create(
+        team=dashboard.team,
+        dashboard=dashboard,
+        name="Sample - Purchase conversion funnel",
+        type=TRENDS_FUNNEL,
+        filters={
+            TREND_FILTER_TYPE_EVENTS: [
+                {"id": "$pageview", "type": TREND_FILTER_TYPE_EVENTS, "order": 0},
+                {
+                    "id": "$autocapture",
+                    "name": "Clicked purchase button",
+                    "type": TREND_FILTER_TYPE_EVENTS,
+                    PROPERTIES: [
+                        {"key": "$event_type", "type": "event", "value": "click"},
+                        {"key": "text", "type": "element", "value": "Purchase"},
+                    ],
+                    "order": 1,
+                },
+                {
+                    "id": "$autocapture",
+                    "name": "Submitted checkout form",
+                    "type": TREND_FILTER_TYPE_EVENTS,
+                    PROPERTIES: [
+                        {"key": "$event_type", "type": "event", "value": "submit"},
+                        {"key": "$pathname", "type": "event", "value": "/purchase"},
+                    ],
+                    "order": 2,
+                },
+                {"id": "Order Completed", "name": "Order Completed", "type": TREND_FILTER_TYPE_EVENTS, "order": 3},
+            ],
+            INSIGHT: "FUNNELS",
+        },
+        last_refresh=now(),
+        color=random.choice(DASHBOARD_COLORS),
+    )
+
+    DashboardItem.objects.create(
+        team=dashboard.team,
+        dashboard=dashboard,
+        name="Users by browser (last 2 weeks)",
+        type=TRENDS_PIE,
+        filters={
+            TREND_FILTER_TYPE_EVENTS: [{"id": "$pageview", "math": "dau", "type": TREND_FILTER_TYPE_EVENTS}],
+            DATE_FROM: "-14d",
+            INTERVAL: "day",
+            BREAKDOWN_TYPE: "person",
+            BREAKDOWN: "$browser",
+            DISPLAY: TRENDS_PIE,
+        },
+        last_refresh=now(),
+    )
+
+    DashboardItem.objects.create(
+        team=dashboard.team,
+        dashboard=dashboard,
+        name="Users by traffic source",
+        type=TRENDS_LINEAR,
+        filters={
+            TREND_FILTER_TYPE_EVENTS: [{"id": "$pageview", "math": "dau", "type": TREND_FILTER_TYPE_EVENTS}],
+            INTERVAL: "day",
+            BREAKDOWN_TYPE: "event",
+            BREAKDOWN: "$initial_referring_domain",
+        },
+        last_refresh=now(),
+    )
```

Comment:
Could maybe decouple dashboard item creation from the dry dashboard templates itself? Not super needed now, but would be more elegant if hydration was something like
```Python
DashboardItem.objects.bulk_create([
    DashboardItem(team=dashboard.team, last_refresh=now(), **item_schematic) for item_template in item_templaes
])
```